pyalarmdotcomajax/pyalarmdotcomajax.py

```diff
@@ -326,10 +379,12 @@ async def async_update(self):
             raise
         return True
 
-    async def _send(self, event, forcebypass, noentrydelay, silentarming):
+    async def _send(self, command, event, forcebypass, noentrydelay, silentarming):

```

Comment:
Does `mode` make more sense than `command`?

Reply:
Was thinking of that, but I didn't want to use the same name as in async_update, as the values aren't the same.

What about `resource` or `controller` ?

Reply:
I'm a bit confused - it's annotated in L387 as lock or partition, but the usage in L405/L410 is "alarm" or "lock" which seems to match the `mode`s in `async_update`. Are these different concepts?

Reply:
You are right. It is the same term (too early in morning, no coffee.

Renamed.